### PR TITLE
feat: add the code

### LINKED ISSUE #1: Make a Casdoor Flutter SDK like other languages
See other languages: https://github.com/casdoor?q=sdk&type=all&language=&sort=

This SDK will be used by Flutter apps to connect to Casdoor authentication platform.

### PR DESCRIPTION
**Pull request by ziliangyu**
Fix: https://github.com/casdoor/casdoor-flutter-sdk/issues/1

### CODE REVIEW

**Comment by hsluoyz** [comment]
@xiaobeiswift plz review

**Comment by hsluoyz** [comment]
@xiaobeiswift plz review

**Review by xiaobeiswift — APPROVED** [review]
plz merge

File: README.md [hunk 1]
```diff
@@ -1 +1,130 @@
...
+The following platforms are supported:
+
++ Android
++ iOS 
++ macos
++ Web
+
+|  **Android**   | **iOS**  | **Web** |
+|  ----  | ----  |----  |
+| ![Android](https://cdn.jsdelivr.net/gh/ziliangyu/picb@main/screen-andriod.gif) |![iOS](https://cdn.jsdelivr.net/gh/ziliangyu/picb@main/screen-ios.gif)  |![Web](https://cdn.jsdelivr.net/gh/ziliangyu/picb@main/screen-web.gif) |
```
Comment (hsluoyz) [inline README.md]: Host these pictures inside this repo

File: android/src/main/kotlin/com/example/casdoor_flutter_sdk/CasdoorFlutterSdkPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package com.example.casdoor_flutter_sdk
```
Comment (hsluoyz) [inline android/src/main/kotlin/com/example/casdoor_flutter_sdk/CasdoorFlutterSdkPlugin.kt:15]: Add Casdoor license headers to all source code files

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**